other/edge-server/client/README.md

```diff
@@ -1,38 +1,116 @@
 # Edge Server with MongoDB Node.js Driver
 
-This example application uses the MongoDB Node.js Driver to connect to 
-Edge Server. It has a React frontend for performing CRUD operations. It uses 
-the MongoDB Node.js Driver in an Express server application to connect to
-the Edge Server.
+This example todo application demonstrates using Edge Server with a MongoDB Driver. It uses the MongoDB Node.js Driver in an Express server application to connect to 
+Edge Server. It has a React frontend for performing CRUD operations. 
 
 React client -> Express server (Node.js Driver) -> Edge Server -> Atlas
 
 To use this example application, you must:
 
-1. Create an App Services App using the backend configuration provided in
-  this project's [TODO: Cory insert directory name here] directory.
+1. Create an App Services App that uses the configured backend provided in 
+  this project's ../backend directory.
```

Comment:
Since we're now using a template app for this, we won't include the backend directory in the artifact repo - so it doesn't make sense to refer to it here. 
```suggestion
1. Create an App Services App based on the `edge-server.todo` template app. Get Edge Server enabled for your new app.
```